src/id3/v2/tag.rs

```diff
@@ -1634,4 +1723,103 @@ mod tests {
 
 		assert_eq!(id3v2, reparsed);
 	}
+
+	#[test]
```

Comment:
The tests only cover a single case. Missing are test for only number and only total.

Reply:
I am also curious to know what happens if the frame value

- is empty (contains only whitespace)
- "/"
- "/1"
- "1/"

This should be handled by lightweight unit tests, no need for heavy-weight file tests.

Reply:
I added tests about the missing tests at first.

I will add tests about the frame values.